internal/record/recording_https_proxy.go

```diff
@@ -112,17 +118,17 @@ func (r *RecordingHTTPSProxy) recordRequest(req *http.Request) (string, error) {
 	recordedRequest.Request = r.redactor.String(recordedRequest.Request)
 	recordedRequest.Body = r.redactor.Bytes(recordedRequest.Body)
 
-	reqHash, err := recordedRequest.ComputeSum()
+	fileName, err := recordedRequest.GetRecordingFileName()
```

Comment:
ditto

Reply:
I use `err` here, do you mean other  change?